models/workitemtype_test.go

```diff
@@ -63,27 +56,22 @@ func TestMarshalEnumType(t *testing.T) {
 		Required: true,
 	}
 
-	timeNow := time.Now()
-	wt := WorkItemType{
-		Lifecycle: Lifecycle{
```

Comment:
@sbose78  There's no need to assign a zero value as it is done automatically by Go ;)